### ISSUE TITLE
Generates report for only one module on multi-module Android project

### ISSUE THREAD

**Issue by lokeshbhattarai** [Kotlin/kotlinx-kover#63]
My project has three modules; `app`, `data` and `uiModule`. `app` has build flavors and has dependency on `data` and `uiModule`.

I have the plugin set up in my root gradle file

```
buildscript {
    repositories {
        mavenCentral()
    }

    dependencies {
        classpath("org.jetbrains.kotlinx:kover:0.4.2")
    }
}

apply(plugin = "kover")

```

When I run `./gradlew koverHtmlReport`, the coverage reports are being generated in build directory for each module but all of them have reports for uiModule packages only. I have unit tests in all three modules.

I'm using gradle version 7.1

Is there anything I am missing?

**Comment by shanshin** [comment]
Hi, may you clarify, did you somehow configured tasks or extensions for Kover?

**Comment by lokeshbhattarai** [comment]
I didn't have any configuration on tasks or any extensions. Let me know if you need any more information.

**Comment by puyunfeng-sl** [comment]
After testing we can get the Kover report in the android project, and secondly we can get the report in the common module of the KMM project, but we can't get the report in the android module of the KMM project.

**Comment by shanshin** [comment]
@lokeshbhattarai, is the issue reproduced in version `0.5.0-RC2`?

**Comment by lokeshbhattarai** [comment]
So I tried version `0.5.0-RC2` and found out that it was still doing the same thing. However, I looked closer into the generated `index.html` file and found out that it's actually generating coverages for each module. It turns out, when I open the `index.html` in the browser (Using "Open in" menu from AndroidStudio) it's opening the file using the path from repository root and so it's always showing report for `uiModule`. Not sure why it picks `uiModule` though. Anyway, when I copy the absolute path of `index.html` file in `app` module's build directory and open it in the browser it shows the right output. 

It's probably not the issue with this plugin or even if it is, it seems a different issue than what I originally reported here. So I think it's safe to close this issue. I will leave this up to you.
